app/scripts/modules/core/cluster/filter/clusterFilter.service.ts

```diff
@@ -469,10 +471,12 @@ export class ClusterFilterService {
 
 }
 
+export let clusterFilterService: ClusterFilterService;
 export const CLUSTER_FILTER_SERVICE = 'spinnaker.core.cluster.filter.service';
 module(CLUSTER_FILTER_SERVICE, [
   require('angular-ui-router').default,
   CLUSTER_FILTER_MODEL,
   require('./multiselect.model'),
   require('../../filterModel/filter.model.service'),
-]).service('clusterFilterService', ClusterFilterService);
+]).service('clusterFilterService', ClusterFilterService)
+  .run(($injector: any) => clusterFilterService = <ClusterFilterService>$injector.get('clusterFilterService'));
```

Comment:
Happy to change the way we make services importable, but if we do, let's do that wholesale in a different PR. I think the ReactInjector way makes sense when we don't already have ng.module.